spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkReadOptions.java

```diff
@@ -90,4 +90,6 @@ private SparkReadOptions() {}
   public static final String VERSION_AS_OF = "versionAsOf";
 
   public static final String TIMESTAMP_AS_OF = "timestampAsOf";
+
+  public static final String AGGREGATE_PUSH_DOWN_ENABLED = "aggregatePushDownEnabled";
```

Comment:
Iceberg uses properties that are words separated by `-`. The reason why the two above use camelCase instead is that they are standard options used by Spark. I think for this we should use `aggregate-push-down-enabled`.

Reply:
+1

Reply:
Fixed. Thanks